main.go

```diff
@@ -36,8 +35,12 @@ type link struct {
 
 // D3 is the structure used to construct the data for the frontend D3 graph.
 type D3 struct {
-	Nodes []node `json:"nodes"`
-	Links []link `json:"links"`
+	Nodes           []node `json:"nodes"`
+	Links           []link `json:"links"`
+	RequestsUsed    int    `json:"requestsUsed"`
```

Comment:
The D3 struct is reserved for graph data, it would be better to have other details stored in a separate struct such as `Dashboard` or `DashboardDetails`.